native/src/main/java/io/ballerina/stdlib/constraint/Constraints.java

```diff
@@ -65,7 +68,7 @@ public static Object validate(Object value, BTypedesc typedesc) {
         } catch (InternalValidationException e) {
             return ErrorUtils.createError(e.getMessage());
         } catch (RuntimeException e) {
```

Comment:
Can we catch BError here return data binding error rather returning generic error?

Reply:
data binding error will be a subtype of constraint:Error

Reply:
Should we call it `constraint:TypeConversionError` rather than `constraint:DataBindingError`?
Since we don't have a reference to data binding in the spec, `TypeConversionError` seems to be simple and understandable. WDYT?

Reply:
I agree as long as it sounds the same

Reply:
Fixed via [90adaf6](https://github.com/ballerina-platform/module-ballerina-constraint/pull/52/commits/90adaf6da2a370cc3cc8dd271bdeccca35edc366)